cypress/e2e/CMS/$Event/index.js

```diff
@@ -0,0 +1,177 @@
+    });
+});
+
+And('the event date is displaying todays date', () => {
+    const months = [
+        'January',
+        'February',
+        'March',
+        'April',
+        'May',
+        'June',
+        'July',
+        'August',
+        'September',
+        'October',
+        'November',
+        'December'
+    ];
+    const date = new Date();
+    const year = date.getFullYear();
+    const month = months[date.getMonth()];
+    const day = date.getDate();
+    const expectedDate = `${month} ${day}, ${year}`;
+    cy.get('p.event-date-time').should('include.text', expectedDate);
+});
+
+And('the event start time displays in the following format {string}', (num) => {
+    cy.get('p.event-date-time').invoke('text').then((time) => {
+        const startAndEnd = time.split('|');
+        const startAndEndtime = startAndEnd[1].split('–');
+        const regex = new RegExp(num);
+        expect(startAndEndtime[0]).to.match(regex);
+    });
+});
+
+And('the event end time displays {string}', (num) => {
+    cy.get('p.event-date-time').should('include.text', num);
+});
+
+And('the Venue location field displays {string}', (locVenue) => {
+    cy.get('span.event-location-venue').should('have.text', locVenue);
+});
+
+And('the room location field displays {string}', (roomNum) => {
+    cy.get('span.event-location-room').should('have.text', roomNum);
+});
+
+And('the city-state location field displays {string}', (cityState) => {
+    cy.get('span.event-location-city-state').should('have.text', cityState);
+});
+
+And('{string} is a link with href that matches {string}', (linkText, linkHref) => {
+    const regex = new RegExp(linkHref);
+    cy.get('div#cgvBody p').find('a').should('have.text', linkText);
+    cy.get('div#cgvBody p').find('a').should('have.attr', 'href').and('match', regex);
+});
+
+And('description reads {string}', (desc) => {
+    cy.get('#cgvBody > :nth-child(6)').should('be.visible')
+});
+
+And('the lead image for event is matching earlier selected image', () => {
+    cy.get('.centered-element > img').then($el => {
+        const source = $el[0].getAttribute('src');
+        const actSrc = source.replace(/\?itok=[\S]+/, '').replace(/^(.*?)\/public/, '')
+        const expectedSrc = (imageSrc.replace(/\?itok=[\S]+/, '')).replace(/^(.*?)\/public/, '')
+        expect(actSrc).to.include(expectedSrc);
+    })
+});
+
+And('user checks {string} checkbox', (targetCheckbox) => {
+    cy.get(`div[class*="field-all-day-event-value"] label:contains("${targetCheckbox}")`).as('checkbox');
+    cy.get('@checkbox').parent().find('input').check();
+});
+
+And('event date does not display time', () => {
+    cy.get('p.event-date-time').should('not.include.text', '|');
+});
+
+And('user removes the Lead Image', () => {
+    cy.get('#edit-field-image-article-current-items-0-remove-button').click({ force: true })
+});
+
+And('the event date displays today as a start and tomorrow as the end of event', () => {
```

Comment:
remove this step since we got rid of it in a feature file

Reply:
removed